attestation-agent/lib/src/token.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use kbs_protocol::{evidence_provider::NativeEvidenceProvider, KbsClientBuilder};
+use serde::Serialize;
+use tokio::fs;
+
+#[derive(Serialize)]
+struct Message {
+    token: String,
+    tee_keypair: String,
+}
+
+pub(crate) async fn get_kbs_token() -> Result<Vec<u8>> {
+    let evidence_provider = Box::new(NativeEvidenceProvider::new()?);
+    let kbs_host_addr = get_kbs_host_from_cmdline().await?;
+    let mut client =
+        KbsClientBuilder::with_evidence_provider(evidence_provider, &kbs_host_addr).build()?;
+
+    let (token, tee_keypair) = client.get_token().await?;
+    let message = Message {
+        token: token.content,
+        tee_keypair: tee_keypair.to_pkcs1_pem()?.to_string(),
```

Comment:
just double confirm, do we need return `tee_keypair`?

Reply:
yes. pubkey is used to encrypt the data of resources